tests/functionaltests/UXTestHelpers.mm

```diff
@@ -106,4 +114,27 @@
     }
 }
 
+//
+// UXEvent methods
+//
+void UXEvent::Signal() {
+    [_condition lock];
+
+    _signaled = true;
+    [_condition signal];
+
+    [_condition unlock];
+}
+
+bool UXEvent::Wait(int timeOutInSeconds) {
```

Comment:
How about WaitFor? WaitFor(3.0f /* timeoutInSeconds */)

---
In reply to: [98268635](https://github.com/Microsoft/WinObjC/pull/1855#discussion_r98268635) [](ancestors = 98268635,98267209)